### PR TITLE
MBTiles: Add WEBP support

### PR DESCRIPTION
**Pull request by drons**

### CODE REVIEW

**Comment by rouault** [comment]
could you add a test for that ?
You can reference #3437 in the commit message

**Comment by drons** [comment]
> could you add a test for that ?

OK, coming soon.

File: autotest/gdrivers/mbtiles.py [hunk 1]
```diff
@@ -598,3 +598,76 @@ def test_mbtiles_create():
...
+        "ZOOM_LEVEL": "11",
+        "minzoom": "11",
+        "maxzoom": "11",
+        "format": "webp",
+        "version": "1.3",
+        "type": "baselayer",
+        "name": "webp_mbtiles_name",
+        "description": "webp_mbtiles_dsc",
+    }
+    print(got_md)
```
Comment (rouault) [inline autotest/gdrivers/mbtiles.py:669]:
```suggestion
```

**Comment by drons** [comment]
Thanks!

**Pull request merged**